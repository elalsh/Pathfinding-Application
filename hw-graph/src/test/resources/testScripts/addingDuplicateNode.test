# Tests adding duplicate node to the graph

# Create a graph
CreateGraph graph1

# Add a pair of nodes A and B
AddNode graph1 A
AddNode graph1 B

# Print the nodes in the graph
ListNodes graph1

# Add duplicate node
AddNode graph1 B

# Print the nodes in the graph
ListNodes graph1