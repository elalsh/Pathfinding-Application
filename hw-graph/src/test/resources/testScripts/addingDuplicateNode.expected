# Tests adding duplicate node to the graph

# Create a graph
created graph graph1

# Add a pair of nodes A and B
added node A to graph1
added node B to graph1

# Print the nodes in the graph
graph1 contains: A B

# Add duplicate node
added node B to graph1

# Print the nodes in the graph
graph1 contains: A B
